### PR TITLE
Alps 1228 - STW WTS for GoPro view

### PR DESCRIPTION
**Pull request by aberthet-gpsw**
Adding the STW and WTS methods for the GoPro view, with unit testing.

I moved the `View.spec.js` file as `ViewRectilinear.spec.js`, so that we have two separated View test file.

Note that the excessive warning we had before is now fixed (bad cleaning of HotspotDOM).

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

File: src/hotspots/HotspotDOM.js [hunk 1]
```diff
@@ -331,6 +331,8 @@ FORGE.HotspotDOM.prototype.destroy = function()
 
     this._clearEvents();
     this._events = null;
+
+    this._unregister();
```
Comment (rroux-gpsw) [inline src/hotspots/HotspotDOM.js]: You'd better do this : ```FORGE.BaseObject.prototype.destroy.call(this);```

Comment (rroux-gpsw) [inline src/hotspots/HotspotDOM.js]: That's part of the BaseObject job

**Comment by bbaudel-gpsw** [comment]
There is always a kind of "bounce" effect on 3d hotspots objects from camera fov approximatively 130 to 175 with GoPro view.

**Comment by aberthet-gpsw** [comment]
@bbaudel-gpsw yep, this issue isn't meant to be resolved in this PR. The bounce occurred during the change in projection distance, so between 90º and 180º

**Pull request merged**